configuration/configuration_test.go

```diff
@@ -219,6 +220,7 @@ func TestReadConfigurationFromFileOverrideEverything(t *testing.T) {
 	test.Equals(t, "Mona", actualConfiguration.TimerUser)
 	test.Equals(t, "https://timer.innoq.io/", actualConfiguration.TimerUrl)
 	test.Equals(t, "team-stash-name", actualConfiguration.StashName)
+	test.Equals(t, "team-stash-name", actualConfiguration.StashName)
```

Comment:
Why is this duplicated here?

Reply:
I think you forgot to check for the `actualConfiguration.StartCISkipCommitMessage`

Reply:
Yep, my bad I'll fix that right away.